app/controllers/answers_controller.rb

```diff
@@ -1,12 +1,36 @@
 class AnswersController < ApplicationController
+  before_action :find_category
   def index
+
   end
 
   def new
-    @category = Category.find(params[:category_id])
-    @word = @category.words.build
-    @choices = choices.all
+    @words =  @category.words.paginate(page:params[:page], per_page: 1)
+    @lesson = Lesson.find(params[:lesson_id])
+    @answer = Answer.new
   end
 
+  def create
+    @lesson = Lesson.find(params[:lesson_id])
+    @answer = Answer.new(answer_params)
+    if @lesson.save
+      unless params[:page].empty?
+        redirect_to new_category_lesson_answer_url(@word.page: params[:page])
```

Comment:
This is the wrong way of calling the ID for the route. Did you forget how to put ID for a route?
It's called `new_category_lesson_answer_url`.

What ID does it need?
1. category
2. lesson

And what did you put?
`(@word.page: params[:page])`

The way you implement pagination is wrong as well.
Did you forget about this?
![image](https://user-images.githubusercontent.com/35232348/105464425-8cc90480-5ccc-11eb-9a43-d3db74a5cb8f.png)

WHAT TO DO:
1. Add the proper ID
2. Implement the pagination properly